### ISSUE TITLE
Steam library extension requiring authentification when API key is used

### ISSUE THREAD

**Issue by Whitechapel-1888** [JosefNemec/PlayniteExtensions#374]
Short description:
Playnite requires extra steam authentification despite added API key

Expected behaviour:
No extra authentification should be required

Detailed description:
I have set up playnite to use the steam API key to log into steam. However, every now and then playnite notifies me that I have to authenticate my steam account.
![image](https://github.com/JosefNemec/PlayniteExtensions/assets/44060684/dcf83222-bdd6-4e24-b987-656c666d3f7f)
No log is generated when clicking on that notification.

Now, I wouldn't mind just pressing the button, but **Steam flagged my account as stolen because playnite needed authentification** while I had it open on my desktop. That is a huge issue and steam accounts can be suspended for this issue. 
![image](https://github.com/JosefNemec/PlayniteExtensions/assets/44060684/6c1ab304-0ccf-4022-922f-4ee5eef3867e)

Please fix this ASAP.
Not trying to be arrogant here, but I really do not want my steam account being suspended due to playnite having the double authentification problem since last year....

**Comment by JosefNemec** [comment]
If you account has been disabled temporarily, it also means that API access will not work as far as I know, you need to unblock your account. 

> Steam flagged my account as stolen because playnite needed authentification

I have never seen Steam account being disabled because you logged into your Steam account via Playnite. We've been using this auth process for years now and this is first time somebody had this issue. Authentication process is done the same way as if you logged in via any other browser, there's no difference from Valve's point of view if your login via browser or Playnite. So I'm 99% confident that this is not Playnite's fault.

**Comment by JosefNemec** [comment]
> double authentification problem

There is no double authentication being done in Playnite. You login once to get your Steam ID and then API is always used, even if you don't use your own API key (our "public" key is used instead). After the initial login and successful Steam ID download, there are no other login attempts, just Steam web API calls.

**Comment by Whitechapel-1888** [comment]
Before my account was flagged, I had this notification pop up multiple times last week.
The plugin success story uses the API key, also (it's stored separately in the extensions settings, so I had playnite configured in two locations to use the API key). Could that be the culprit?

I unfortunately didn't keep track of the time I used playnite last Friday, but I am pretty sure that this happened due to my system logging into steam in different ways (I had to log into the website, playnite complained, and I use the mobile authenticator, too).
